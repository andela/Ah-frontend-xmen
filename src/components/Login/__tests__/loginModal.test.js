import { shallow } from 'enzyme';
import React from 'react';
import LoginModal from '../loginModal';

const props = {
  onClose: jest.fn(),
  open: false,
};
describe('login modal', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<LoginModal {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
