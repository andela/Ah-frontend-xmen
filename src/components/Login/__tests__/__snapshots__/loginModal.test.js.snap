// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login modal should render without crashing 1`] = `ShallowWrapper {}`;
